charts/agent-plane/templates/_helpers.tpl

```diff
@@ -1,9 +1,9 @@
 #
-#  Copyright (c) 2023 T-Systems International GmbH
+#  Copyright (c) 2023,2024 T-Systems International GmbH
 #  Copyright (c) 2023 ZF Friedrichshafen AG
 #  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
 #  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
-#  Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+#  Copyright (c) 2021, 2024 Contributors to the Eclipse Foundation
```

Comment:
Same here, please check all the files in the PR to have a proper license header format.

Reply:
went over the complete directory using a search expression. Hope I got all

Reply:
had to mitigate some SAST issues at the same time (and did the same also for the https://github.com/eclipse-tractusx/knowledge-agents/pull/120